frame/staking/src/pallet/impls.rs

```diff
@@ -669,17 +666,34 @@ impl<T: Config> Pallet<T> {
 
 			if !targets.is_empty() {
 				let vote_weight = weight_of(&nominator);
-				all_voters.push((nominator, vote_weight, targets))
+				all_voters.push((nominator, vote_weight, targets));
+				nominator_count.saturating_inc();
 			}
 		}
 
+		Self::register_weight(T::WeightInfo::get_npos_voters(
+			validator_count,
+			nominator_count,
+			slashing_spans.len() as u32,
+		));
+
 		all_voters
 	}
 
 	/// This is a very expensive function and result should be cached versus being called multiple
 	/// times.
 	pub fn get_npos_targets() -> Vec<T::AccountId> {
```

Comment:
Is it worth noting in the doc comment that these calls `register_weight` and thus adds weight as part of mandatory dispatch?